src/Components/Loader/Loader.jsx

```diff
@@ -0,0 +1,7 @@
+import './Loader.css'
+
+export const Loader = () => {
+    return (
+        <div className='circle'></div>
+    )
```

Comment:
appreciation - good that you are showing loader

Reply:
Thanks!